rust/utils/run-locally/src/cosmos/mod.rs

```diff
@@ -57,8 +57,8 @@ fn default_keys<'a>() -> [(&'a str, &'a str); 6] {
     ]
 }
 
-const CW_HYPERLANE_GIT: &str = "https://github.com/many-things/cw-hyperlane";
-const CW_HYPERLANE_VERSION: &str = "0.0.6-rc6";
+const CW_HYPERLANE_GIT: &str = "https://github.com/hyperlane-xyz/cosmwasm";
+const CW_HYPERLANE_VERSION: &str = "injective-testnet";
```

Comment:
Is that our versioning scheme now?

Reply:
idk what the other one was :P
this made it easier to follow